### ISSUE TITLE
Error in uptake_list_school target

### ISSUE THREAD

**Issue by atumscott** [nptscot/npt#171]
With the following error:

```
>     p = "balanced"
>       message("Uptake for ", p, " school routes")
Uptake for balanced school routes
>       routes = r_school[[p]] %>%
+         get_scenario_go_dutch(purpose = "school") %>%
+         as_tibble()
Error in `stopifnot()`:
ℹ In argument: `pcycle_go_dutch = pcycle_go_dutch + (bicycle/all)`.
ℹ In group 1: `route_id = 1`.
Caused by error:
! object 'bicycle' not found
Run `rlang::last_trace()` to see where the error occurred.
```

**Comment by Robinlovelace** [comment]
Issue is well-illustrated by this:


```
waldo::compare(names(r_school[[1]]), names(r_school[[2]]))
     old             | new                      
 [1] "geo_code1"     - "DataZone" [1]           
 [2] "geo_code2"     - "SeedCode" [2]           
 [3] "all"           - "count"    [3]           
 [4] "from_home"     -                          
 [5] "train"         -                          
 [6] "bus"           -                          
 [7] "car_driver"    -                          
 [8] "car_passenger" -                          
 [9] "bicycle"       -                          
[10] "foot"          -                          
 ... ...               ...        and 7 more ...
```

**Comment by Robinlovelace** [comment]
Heads-up also @joeytalbot. This was a consequence of fixing a previous regression that was fixed in #170 .
